README.md

```diff
@@ -80,7 +80,7 @@ label-studio
 ### Install locally with Anaconda
 
 ```bash
-conda create --name label-studio
+conda create --name label-studio python=3.7
```

Comment:
why should we specify this version of python? maybe we can use a range from 3.7 to 3.10?

Reply:
3.7 will be deprecated in 2 months